apps/web/src/services/asset-inventory/composables/use-metric-example-list-query.ts

```diff
@@ -0,0 +1,27 @@
+import { computed, type ComputedRef } from 'vue';
+
+import { useMetricExampleApi } from '@/api-clients/inventory/metric-example/composables/use-metric-example-api';
+import type { MetricExampleListParameters } from '@/api-clients/inventory/metric-example/schema/api-verbs/list';
+import { useServiceQueryKey } from '@/query/core/query-key/use-service-query-key';
+import { useScopedQuery } from '@/query/service-query/use-scoped-query';
+
+interface UseMetricExampleListQueryOptions {
+    params: ComputedRef<MetricExampleListParameters>;
+}
+
+export const useMetricExampleListQuery = ({
+    params,
+}: UseMetricExampleListQueryOptions) => {
```

Comment:
Suggestion:
```suggestion

export const useMetricExampleListQuery = ({
    params,
}: UseMetricExampleListQueryOptions = {}) => {
```

then.. you can write
```
const query = useMetricExampleListQuery();
```
this!

Reply:
ok